src/enhancer.js

```diff
@@ -92,27 +66,26 @@ export default function enhanceWithRadium(
   const component: Function = configOrComposedComponent;
   let ComposedComponent: constructor = component;
 
-  // Handle Native ES classes.
+  // Radium is transpiled in npm, so it isn't really using es6 classes at
+  // runtime.  However, the user of Radium might be.  In this case we have
+  // to maintain forward compatibility with native es classes.
   if (isNativeClass(ComposedComponent)) {
-    // Manually approximate babel's class transpilation, but _with_ a real `new` call.
     ComposedComponent = (function(OrigComponent): constructor {
       function NewComponent() {
-        // Ordinarily, babel would produce something like:
-        //
-        // ```
-        // return _possibleConstructorReturn(this, OrigComponent.apply(this, arguments));
-        // ```
-        //
-        // Instead, we just call `new` directly without the `_possibleConstructorReturn` wrapper.
-        const source = new OrigComponent(...arguments);
-
-        // Then we manually update context with properties.
-        copyProperties(source, this);
-
-        return this;
+        // Use Reflect.construct to simulate 'new'
+        const obj = Reflect.construct(
```

Comment:
What are the implications of using `Reflect` in regards to cross-browser support? I believe IE11 is the only browser that will need a polyfill, would you be able to add a note in the README about including a polyfill like https://github.com/zloirock/core-js#ecmascript-reflect?

A note in this section would probably be best - https://github.com/FormidableLabs/radium#importing-radium

Reply:
@kylecesmat I don't think you can create a "native class" in IE11, so this code would never run in that browser.